add-on/chrome/bootstrap.js

```diff
@@ -172,6 +174,46 @@ var WindowListener = {
       },
 
       /**
+       * Method to send telemetry event when the user opens panel
+       * and hasn't been send yet.
+       */
+      _shouldSendMauTelemetry: function() {
+        this.LoopAPI.sendMessageToHandler({
+          name: "GetLoopPref",
+          data: ["mau.openPanel"]
+        }, pref => {
+          if (!pref) {
```

Comment:
This doesn't really address the 'one month ago' issue... What I'd recommend is to add logic to the 'TelemetryAddValue' message handler to deal with MAU events:

1. Each area of the UI that needs measuring will be dumb and call `TelemetryAddValue`.
2. `TelemetryAddValue` sees an incoming request for 'LOOP_MAU'...
3. ...so it reads the current integer value of the corresponding pref
4. Compares nowDate - prefDate >= 30days
5. If so, we're all good and telemetry value is added. The pref is set to `nowDate` after that, so we won't record that event for another 30 days at least.

What do you think?